### ISSUE TITLE
[Feature-Request] Throughput Monitor on 1.20.1

### ISSUE THREAD

**Issue by PrydeFall** [pedroksl/AdvancedAE#28]
Just wondering if this is going to be implemented into 1.20.1 anytime soon, I have been looking for this exact thing for so long, trying to figure out how to do it with different mods. This would solve all of those issues. I just installed the mod in my ATM9 world to get the quantum computer, I would love to have the throughput monitor as well.

Thank you.

**Comment by pedroksl** [comment]
This has been asked several times in different ways, but you’re basically requesting a backport.

I have done this once to bring the quantum computer and it was quite messy, gave me a bunch of bugs to fix and after that every time I fixed something on the newer version I had to go back to the previous and do it there.

I’m not discarding a port in the future but I’m still developing the mod. This means it’s not stable by any means and fixes are frequent. I’m only one guy doing this as a hobby. There’s also the added issue that 1.20.1 uses forge instead of neo forge, which makes porting a little bit harder.

tldr: I could see myself doing this after I’m mostly done with the mod to avoid having to maintain and fix two versions at once, I hope that’s understandable.

**Comment by PrydeFall** [comment]
no problem, thank you
